tests/integration/cairo1_compiler_bindings/library_functions_test.py

```diff
@@ -23,6 +23,11 @@ def get_mock_for_lib_func(
         return_value = type(
             "return_value", (object,), {"err_code": err_code, "ok": ok}
         )()
+    elif lib_func_name == "declare_legacy":
```

Comment:
why don't we do:
```
if lib_func_name == "declare" or lib_func_name == "declare_legacy":
```
those ret values are the same, right?

Reply:
[17b279e](https://github.com/software-mansion/protostar/pull/1622/commits/17b279e39d758f236aae615516011c107bf59d10)